app/javascript/tests/public_health/actions/UpdateCaseStatus.test.js

```diff
@@ -0,0 +1,246 @@
+      expect(wrapper.state('case_status')).toEqual(case_status_option);
+      expect(wrapper.state('allSelectedAreClosed')).toBeTruthy();
+
+      // When all are closed, there is no follow-up dropdown, so this shoud be 3, instead of the usual 4
+      expect(wrapper.find('FormControl').length).toEqual(3);
+
+      const monitoringReasonOptions = [''].concat(mockMonitoringReasons);
+      wrapper
+        .find('#monitoring_reason')
+        .find('option')
+        .forEach((option, index) => {
+          expect(option.text()).toEqual(monitoringReasonOptions[Number(index)]);
+          wrapper.find('#monitoring_reason').simulate('change', { target: { id: 'monitoring_reason', value: option }, persist: jest.fn() });
+          expect(wrapper.state('monitoring_reason')).toEqual(option);
+        });
+      const mockReasoning = 'I Shall Call Him Squishy And He Shall Be Mine And He Shall Be My Squishy.';
+      wrapper
+        .find('FormGroup')
+        .at(1)
+        .find('FormControl')
+        .simulate('change', { target: { id: 'reasoning', value: mockReasoning }, persist: jest.fn() });
+      expect(wrapper.state('reasoning')).toEqual(mockReasoning);
+      expect(wrapper.find('.character-limit-text').first().text()).toContain(`${2000 - mockReasoning.length} characters remaining`);
+    });
+  });
+
+  it('Properly sets the correct follow-up options for "Suspect", "Unknown, and "Not a Case"', () => {
+    // have to `mount` the wrapper so the event object is properly created when simulating change below
+    const wrapper = getWrapper([mockPatient1, mockPatient2, mockPatient3, mockPatient4]);
+    // "Suspect", "Unknown" and "Not a Case" have identical behavior in the modal
+    ['Suspect', 'Unknown', 'Not a Case'].forEach(case_status_option => {
+      wrapper
+        .find('FormControl')
+        .first()
+        .simulate('change', { target: { id: 'case_status', type: 'change', value: case_status_option }, persist: jest.fn() });
+
+      expect(wrapper.find('p').at(1).text()).toContain('The selected cases will be moved from the isolation workflow to the exposure workflow and placed in the symptomatic, non-reporting, or asymptomatic line list as appropriate.');
+      // there are no dropdowns for "Suspect", "Unknown" and "Not a Case"
+      expect(wrapper.find('FormControl').at(1).exists()).toBeFalsy();
+      wrapper.find('FormCheck').simulate('change', { target: { id: 'apply_to_household', type: 'checkbox', checked: false }, persist: jest.fn() });
+      expect(wrapper.state('apply_to_household')).toBeFalsy();
+      wrapper.find('FormCheck').simulate('change', { target: { id: 'apply_to_household', type: 'checkbox', checked: true }, persist: jest.fn() });
+      expect(wrapper.state('apply_to_household')).toBeTruthy();
+    });
+  });
+
+  it('Properly calls the close method', () => {
+    const wrapper = getWrapper([mockPatient1, mockPatient2, mockPatient3, mockPatient4]);
+
```

Comment:
another nitpick but don't need this space

Reply:
fixed